ping.go

```diff
@@ -633,7 +641,11 @@ func (p *Pinger) sendICMP(conn *icmp.PacketConn) error {
 		dst = &net.UDPAddr{IP: p.ipaddr.IP, Zone: p.ipaddr.Zone}
 	}
 
-	t := append(timeToBytes(time.Now()), uintToBytes(p.Tracker)...)
+	currentUUID, err := p.trackerUUIDs[len(p.trackerUUIDs)-1].MarshalBinary()
+	if err != nil {
+		return fmt.Errorf("unable to marshal UUID binary: %w", err)
+	}
+	t := append(timeToBytes(time.Now()), currentUUID...)
```

Comment:
this bit of logic is reproduced in multiple locations. Should it be moved to a separate function, so that it's easier to extend in the future?

Reply:
Hmm, possibly. Most of the duplication is in the tests. I'm also planning to make changes after this PR to start implementing the tracker + sequence number wrapping.

I'd say let's leave it for later refactoring.

Reply:
cool, thanks!